frontend/src/router.js

```diff
@@ -243,36 +250,30 @@ export default new Router({
           props: (route) => ({ camp: campFromRoute(route) }),
         },
         {
-          path: 'story',
-          name: 'camp/story',
+          path: 'story/period/:periodId/:periodTitle?',
+          name: 'camp/story/period',
```

Comment:
```suggestion
          name: 'camp/period/story',
```
Wie auch bei den anderen Namen.

Reply:
[acad08d](https://github.com/ecamp/ecamp3/pull/3038/commits/acad08ddce9ea3f89ea2ac3b9fb2bdc9de318397)